modules/ROOT/pages/shared-resources.adoc

```diff
@@ -53,11 +53,12 @@ You can find the following files when you package a domain into a JAR file:
 
```

Comment:
KIM NOTES:
It's just one file, please focus only on the green text (new content).